content/en/docs/concepts/storage/windows-storage.md

```diff
@@ -57,9 +57,9 @@ Volume management components are shipped as Kubernetes volume
 [plugin](/docs/concepts/storage/volumes/#types-of-volumes).
 The following broad classes of Kubernetes volume plugins are supported on Windows:
 
-* [`FlexVolume plugins`](/docs/concepts/storage/volumes/#flexvolume)
+* [`FlexVolume` plugins](/docs/concepts/storage/volumes/#flexvolume)
```

Comment:
I'm afraid that the original text, not in monospace, is more appropriate. Have a look at the section about FlexVolume to see the capitalization and font conventions we typically adopt.